File: totosuki/ABC/B/113.py
N = int(input())
T, A = map(int, input().split())
H = list(map(int, input().split()))
H_abs = []
for h in H:
  tmp = T - (h * 0.006)
  H_abs.append(abs(A - tmp))
print(H_abs.index(min(H_abs)) + 1)

File: OCH_Tao/ABC/B/113.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const [T, A] = (input[1].split(" ")).map(x => parseInt(x, 10));
const H = (input[2].split(" ")).map(x => parseInt(x, 10));
let L = [];
for (const X of H) {
  L.push(Math.abs(A - (T - X * 0.006)));
}
console.log(L.indexOf(Math.min(...L)) + 1);